pkg/build/pipelines/ruby/test.yaml

```diff
@@ -0,0 +1,14 @@
+name: Test a Ruby package
```

Comment:
I think we should get rid of this. We're not doing anything inherently specific to Ruby

Reply:
It seems we have `name:` field defined on each pipeline, do you say we should omit this field?

Reply:
Sorry, the pipeline. It looks like it does the same thing a `runs` would do

Reply:
so drop this?